modules/identity/src/Volo.Abp.Identity.Blazor/AbpIdentityBlazorMessageLocalizerExtensions.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Blazorise;
+using Microsoft.Extensions.Localization;
+
+namespace Volo.Abp.Identity.Blazor
+{
+    public static class AbpIdentityBlazorMessageLocalizerExtensions
+    {
+        public static string Localize<T>(this IStringLocalizer<T> stringLocalizer, string format, IEnumerable<string> arguments)
```

Comment:
The reason why I wanted it as an extension method id because it's easier to write `L.Localize` then something like `BlazorMessageLocalizer.Localize(stringLocalizer, message, args)`

Reply:
Created AbpIdentityBlazorMessageLocalizerHelper instead of extension methods